src/compiler/scala/tools/nsc/typechecker/Namers.scala

```diff
@@ -1461,12 +1461,17 @@ trait Namers extends MethodSynthesis {
         // Note that matching MethodType of NullaryMethodType must be nilary not nelary.
         def overriddenNilary(sym: Symbol) = sym.info.isInstanceOf[MethodType]
         if (overridden != NoSymbol && vparamSymss.isEmpty && overridden.alternatives.exists(overriddenNilary)) {
-          if (settings.warnNullaryOverride)
-            context.unit.toCheck += {() =>
-              def javaDetermined(sym: Symbol) = sym.isJavaDefined || isUniversalMember(sym)
-              if (!meth.overrides.exists(javaDetermined))
-                context.warning(ddef.pos, "nullary method assumes an empty parameter list from the overridden definition", WarningCategory.LintNullaryOverride)
-            }
+          def exempt() = meth.overrides.exists(sym => sym.isJavaDefined || isUniversalMember(sym))
+          val msg = "method without a parameter list overrides a method without params"
```

Comment:
```suggestion
          val msg = "method without a parameter list overrides a method with an empty parameter list"
```

Reply:
From Seth's edit in #8814, I added a "single" to the messages.